.github/workflows/main.workflow.yml

```diff
@@ -90,7 +90,7 @@ jobs:
 
       - name: Installing libusb
         run: sudo apt-get install libusb-1.0-0-dev
-        
+
       - name: Installing DepthAi Core
         run: sudo wget -qO- https://raw.githubusercontent.com/luxonis/depthai-ros/noetic-devel/install_dependencies.sh | sudo bash
```

Comment:
Might make sense to put the branch name here instead (or commit hash directly) as:
```
sudo wget -qO- https://raw.githubusercontent.com/luxonis/depthai-ros/$GITHUB_SHA/install_dependencies.sh | sudo bash
```
So any changes to this file are tested by CI as well. (or set to the current branch)

Reply:
Makes sense. Will do.